cluster/charts/rook-minio-objectstore/templates/objectstore.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: minio.rook.io/v1alpha1
+kind: ObjectStore
+metadata:
+  name: {{ template "rook-minio-objectstore.fullname" . }}
+  labels:
+    app: {{ template "rook-minio-objectstore.name" . }}
+    chart: {{ template "rook-minio-objectstore.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  scope:
+    nodeCount: 4
+  placement:
+    tolerations:
+    nodeAffinity:
+    podAffinity:
+    podAnyAffinity:
+  port: {{ .Values.service.port }}
+  credentials:
+    name: {{ template "rook-minio-objectstore.fullname" . }}
+  storageAmount: "10G"
```

Comment:
This is now a PVC list (https://rook.io/docs/rook/v0.9/minio-object-store-crd.html#sample) so the user can put one or more PVCs.
This should also be configurable by the user.